src/main/java/se/liu/ida/hefquin/cli/modules/ModFederation.java

```diff
@@ -89,147 +88,151 @@ public FederationCatalog getFederationCatalog() {
 	protected void parseFedDescr( final String filename ) {
 		final Model fd = RDFDataMgr.loadModel(filename);
 
-		final ResIterator itSPARQL = fd.listResourcesWithProperty(RDF.type, FD.SPARQLEndpointInterface);
-		while ( itSPARQL.hasNext() ) {
-			final Resource iface = itSPARQL.next();
-			final RDFNode addr = fd.getRequiredProperty(iface, FD.endpointAddress).getObject();
-
-			final String addrStr;
-			if ( addr.isLiteral() ) {
-				addrStr = addr.asLiteral().getLexicalForm();
-			}
-			else if ( addr.isURIResource() ) {
-				addrStr = addr.asResource().getURI();
-			}
-			else {
-				throw new IllegalArgumentException();
-			}
-
-			addSPARQLEndpoint(addrStr);
-		}
-
-		final ResIterator itTPF = fd.listResourcesWithProperty(RDF.type, FD.TPFInterface);
-		while ( itTPF.hasNext() ) {
-			final Resource iface = itTPF.next();
-			final RDFNode addr = fd.getRequiredProperty(iface, FD.exampleFragmentAddress).getObject();
-
-			final String addrStr;
-			if ( addr.isLiteral() ) {
-				addrStr = addr.asLiteral().getLexicalForm();
-			}
-			else if ( addr.isURIResource() ) {
-				addrStr = addr.asResource().getURI();
-			}
-			else {
-				throw new IllegalArgumentException();
-			}
-
-			addTPFServer(addrStr);
-		}
-
-		final ResIterator itBRTPF = fd.listResourcesWithProperty(RDF.type, FD.brTPFInterface);
-		while ( itBRTPF.hasNext() ) {
-			final Resource iface = itBRTPF.next();
-			final RDFNode addr = fd.getRequiredProperty(iface, FD.exampleFragmentAddress).getObject();
-
-			final String addrStr;
-			if ( addr.isLiteral() ) {
-				addrStr = addr.asLiteral().getLexicalForm();
-			}
-			else if ( addr.isURIResource() ) {
-				addrStr = addr.asResource().getURI();
-			}
-			else {
-				throw new IllegalArgumentException();
+		final ResIterator isFedMember = fd.listResourcesWithProperty(RDF.type, FD.FederationMember);
```

Comment:
Why do you call this variable `isFedMember`? In particular, what is the meaning of `is`?

Reply:
This variable is renamed as 'fedMembers'